modules/web/webpack.config.js

```diff
@@ -77,10 +78,23 @@ const config = [{
             $: 'jquery',
             jQuery: 'jquery',
         }),
+        /*
+        new CircularDependencyPlugin({
+            exclude: /a\.css|node_modules/,
+            failOnError: true,
+        }),
+        */
```

Comment:
for the future to find the circular dependencies ... lets keep it for the moment.